protostar/starknet/cheatable_cached_state.py

```diff
@@ -9,8 +9,11 @@
 
 from protostar.starknet.cheaters import BlockInfoCheater, Cheaters
 from protostar.starknet.types import AddressType, ClassHashType, SelectorType
+from protostar.starknet.data_transformer import CairoOrPythonData
 
 
+CallData = list[int]
+
 # pylint: disable=too-many-instance-attributes
 class CheatableCachedState(CachedState):
```

Comment:
I think `_copy` and `_apply` methods need to be updated.
cc @MaksymilianDemitraszek

Reply:
[update methods of CheatableCachedState](https://github.com/software-mansion/protostar/pull/1060/commits/2989b46e2a587679ab8026f90b30ab74c71a1283)